src/less/listbox/ds6/listbox-base.less

```diff
@@ -0,0 +1,78 @@
+@listbox-height: 40px;
+
+.listbox {
+    display: inline-block;
+    box-sizing: border-box;
+    width: 100%;
+
+    &:hover > .listbox__control > input {
+        border-color: @ds6-color-p004-blue;
+    }
+}
+
+.listbox > .listbox__control > input {
```

Comment:
I think this should be just `.listbox__control` now? And the same in all other occurences of `.listbox > .listbox__control`.

Reply:
As you are out today, I can take care of this change and get this merged.